ec/ecresource/deploymentresource/elasticsearch/v2/elasticsearch_config.go

```diff
@@ -44,6 +44,7 @@ func readElasticsearchConfig(in *models.ElasticsearchConfiguration) (*Elasticsea
 		return &ElasticsearchConfig{}, nil
 	}
 
+	config.Plugins = []string{}
```

Comment:
It can lead to the inconsistency error if config is empty. Also it's not needed - `append` can append to `nil`.

Reply:
I mean if TF configuration doesn't have `config`.